core-plugins/src/main/java/io/cdap/plugin/batch/sink/TableSink.java

```diff
@@ -73,13 +73,17 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     Schema outputSchema =
       SchemaValidator.validateOutputSchemaAndInputSchemaIfPresent(tableSinkConfig.getSchemaStr(),
                                                                   tableSinkConfig.getRowField(), pipelineConfigurer);
-    if ((outputSchema != null) && (outputSchema.getFields().size() == 1)) {
+    if (outputSchema != null) {
       String fieldName = outputSchema.getFields().get(0).getName();
-      if (fieldName.equals(tableSinkConfig.getRowField())) {
+      if (outputSchema.getFields().size() == 1 && fieldName.equals(tableSinkConfig.getRowField())) {
         throw new IllegalArgumentException(
-          String.format("Output schema should have columns other than rowkey."));
+            String.format("Output schema should have columns other than rowkey."));
```

Comment:
do we need `String.format`? Can't it just be `throw new IllegalArgument("Output schema should have columns other than rowkey.")`?

Reply:
Don't think so. I assumed it was convention, but it seems like it's left out other places. Fixing this.

Thanks!